libs/solid/profile/src/lib/components/base/base.component.html

```diff
@@ -89,15 +84,21 @@
         SelectedProfile?.trivial_name
       }}</span>
     </div>
-    <button (click)="swipeLeft()" [disabled]="SwipeLeft < 0" mat-icon-button>
-      <mat-icon>navigate_before</mat-icon>
-    </button>
-    <button (click)="selectProfile()" mat-icon-button>
-      <mat-icon>{{ SplitLayout ? 'close' : 'expand_less' }}</mat-icon>
-    </button>
-    <button (click)="swipeRight()" [disabled]="SwipeRight < 0" mat-icon-button>
-      <mat-icon>navigate_next</mat-icon>
-    </button>
+    <div id="profile-detail-toolbar">
```

Comment:
wollen wir das noch umbenennen in `profile-detail-navigation`?

Reply:
Selbstverständlich !